ginivision/src/androidTest/java/net/gini/android/vision/test/exif/ExifReaderException.java

```diff
@@ -0,0 +1,14 @@
+package net.gini.android.vision.test.exif;
+
+import android.support.annotation.NonNull;
+
+public class ExifReaderException extends RuntimeException {
```

Comment:
Why do we need `ExifReaderException `. It doesn't seem to extend anything from `RuntimeException `. It's basically the same thing with a different name.